components/admin_console/message_export_settings.jsx

```diff
@@ -60,6 +60,21 @@ export default class MessageExportSettings extends AdminSettings {
         return state;
     }
 
+    getJobDetails = (job) => {
+        if (job.data && job.data.messages_exported) {
+            return (
+                <FormattedMessage
+                    id='admin.complianceExport.messagesExportedCount'
+                    defaultMessage='{count} messages exported.'
+                    values={{
+                        count: job.data.messages_exported,
+                    }}
+                />
+            );
+        }
+        return null;
+    };
```

Comment:
A reminder that you can use object destructuring in a function parameter like this:
```js
    getJobDetails = ({data}) => {
        if (data && data.messages_exported) {
            return (
                <FormattedMessage
                    id='admin.complianceExport.messagesExportedCount'
                    defaultMessage='{count} messages exported.'
                    values={{
                        count: data.messages_exported,
                    }}
                />
            );
        }
        return null;
    };
```